### ISSUE TITLE
Dockerfile not build

### ISSUE THREAD

**Issue by arquio** [APIs-guru/graphql-faker#42]
``` bash
$ git clone https://github.com/APIs-guru/graphql-faker.git
$ cd graphql-faker
$ docker build -t graphql-faker  .
```
```
Sending build context to Docker daemon  4.304MB
Step 1/6 : FROM node:8.8.1-alpine
8.8.1-alpine: Pulling from library/node
b56ae66c2937: Pull complete 
16e68ed80c6a: Pull complete 
4818769bd05b: Pull complete 
Digest: sha256:ab9161b1be7ed90f1bcf8d483ef72f138010054732bb12a3288ec679a41f2226
Status: Downloaded newer image for node:8.8.1-alpine
 ---> 04a3ba95f191
Step 2/6 : MAINTAINER James Kyburz "james.kyburz@gmail.com"
 ---> Running in 5d8869c5a605
 ---> 8c406551f0af
Removing intermediate container 5d8869c5a605
Step 3/6 : RUN apk --no-cache add --virtual native-deps   git &&   npm install graphql-faker -g &&  npm cache clean --force &&  apk del native-deps
 ---> Running in 264c7cee5faa
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/main/x86_64/APKINDEX.tar.gz
WARNING: Ignoring http://dl-cdn.alpinelinux.org/alpine/v3.6/main/x86_64/APKINDEX.tar.gz: could not connect to server (check repositories file)
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/community/x86_64/APKINDEX.tar.gz
WARNING: Ignoring http://dl-cdn.alpinelinux.org/alpine/v3.6/community/x86_64/APKINDEX.tar.gz: could not connect to server (check repositories file)
ERROR: unsatisfiable constraints:
  native-deps-0:
    masked in: cache
    satisfies: world[native-deps]
  git (missing):
    required by:
The command '/bin/sh -c apk --no-cache add --virtual native-deps   git &&   npm install graphql-faker -g &&  npm cache clean --force &&  apk del native-deps' returned a non-zero code: 2
```

**Comment by arquio** [comment]
Related https://github.com/nodejs/docker-node/pull/367

**Comment by IvanGoncharov** [comment]
@arquio I don't use Docker myself and `Dockerfile` was in community PR I merged.
Can you please explain in more details what's wrong with `Dockerfile`?
If you know how to fix it PR would be highly welcome ❤️

**Comment by arquio** [comment]
@IvanGoncharov I'll try when be solved the problem with the simple image "alpine".

At the moment, I did a very simple Dockerfile for us labX in https://github.com/travelgateX/labX-graphql-benchmarks/tree/master/team-resources/task2-server

Thanks for all!

**Comment by spawnia** [comment]
This should be resolved through #44. Also, you do not have to build the image yourself, there is an automated build at https://hub.docker.com/r/apisguru/graphql-faker/

**Comment by IvanGoncharov** [comment]
Thanks @spawnia 
@arquio I will close this issue but feel free to reopen if you still have this problem.
